src/algokit/core/tasks/wallet.py

```diff
@@ -0,0 +1,165 @@
+import json
+import logging
+from dataclasses import dataclass
+
+import keyring
+
+logger = logging.getLogger(__name__)
+
+WALLET_ALIAS_KEYRING_NAMESPACE = "algokit_alias"
+WALLET_ALIASES_KEYRING_NAMESPACE = "algokit_aliases"
+WALLET_ALIASES_KEYRING_USERNAME = "aliases"
+WALLET_ALIASING_MAX_LIMIT = 50
```

Comment:
Minor: Worth including a description of why this is this number? I assume it's a keyring limitation?